CiteSeer.js

```diff
@@ -40,7 +40,8 @@ function detectWeb(doc, url) {
 	if ((url.indexOf('/search') != -1 || url.indexOf('/showciting') != -1) && getSearchResults(doc).length) {
 		return "multiple";
 	}
-	if (url.indexOf('/viewdoc/') != -1 && doc.getElementById('bibtex')) {
+	if ((url.indexOf('/viewdoc/') != -1 && doc.getElementById('bibtex'))
+		|| url.indexOf('/download?doi=') != -1) {
```

Comment:
We prefer nowadays the function `.includes()` for such tests. Can you change these two (and possibly also the ones in the line above)?